README.md

```diff
@@ -23,140 +25,158 @@
 
-* put `export KUBECONFIG=[path-to-repo]/content-k8s-auth-setup/kubeconfig`  in `~/.zshrc`
-* execute `source cluster-login.zsh udde`
+- Upload the binaries and the cluster-login.sh on the release
 
-### Only if you are connecting to EKS
+### How to use locally
 
-When provisioning a new EKS cluster the provisioning script will upload the corresponding kubeconfig
-to S3 bucket named "upp-kubeconfig-ACCOUNT-NUMBER". In order to use it you need to get this kubeconifg and merge it
-with the kubeconfig files of the rest of EKS clusters. This way you will have a single united
-kubeconfig for all the EKS clusters. Inside `update-eks-kubeconfig` directory there is a script
-named `update-eks-kubeconfig.sh`. The script will do this merge for you and will store the merged
-eks-kubecofig under $HOME/.kube
-Connect to the Restricted VPN and execute:
+- rename binary to kubectl-login and put in on your PATH
+- run `source ./cluster-login.sh  cluster-x` or `. ./cluster-login.sh  cluster-x`
 
-```shell
-cd update-eks-kubeconfig/
-bash update-eks-kubeconfig.sh
-```
+#### How to [Fish](https://fishshell.com/) locally
 
-If you want create an alias to easily change the KUBECONFIG:
+- put the following lines in `~/.config/fish/config.fish`:
+  - `set -x KUBECONFIG <PATH_TO membership-developer-setup>/kubeconfig`
+  - `alias k8s-login="source <PATH_TO kubectl-login>/cluster-login.fish $argv"`
+- Use with `k8s-login` (or whatever name you alias for the command)
 
-```shell
-alias export-eks-kubeconfig="export KUBECONFIG=/$HOME/.kube/eks-kubeconfig"
-```
+#### How to use with ZSH
 
-The script `ops-eks-kubeconfig.sh` inside `update-eks-kubeconfig` directory is intended to be used
-from OPS on the jumpbox (Upp Jumpbox p). The script sits in `/usr/local/bin` and is invoked by
-`/etc/skel/.bashrc` everytime a user is connected. The purpose of the script is similar - it will download and
-merge the kubeconfigs for OPS and will get `kubectx` tool on the jumpbox.
+- put `export KUBECONFIG=[path-to-repo]/content-k8s-auth-setup/kubeconfig`  in `~/.zshrc`
+- execute `source cluster-login.zsh udde`
 
-#### Step-by-Step guide how to connect to EKS clusters
+## EKS
 
-1. Connect to Restricted VPN
-1. Checkout the kubectl-login repo
+### Info
 
-```shell
-git clone git@github.com:Financial-Times/kubectl-login.git
-```
+When provisioning a new EKS cluster the provisioning script will upload the
+corresponding kubeconfig to S3 bucket named `upp-kubeconfig-ACCOUNT-NUMBER`.
+In order to use it you need to get this kubeconfig and merge it with the
+kubeconfig files of the rest of EKS clusters. This way you will have a single
+united kubeconfig for all the EKS clusters.
 
-1. Get in `update-eks-kubeconfig/` folder
+If you want to do that automatically, there is this script
+`./update-eks-kubeconfig/update-eks-kubeconfig.sh`.
+The script will do the merge for you. It will store the merged
+`eks-kubecofig` under `$HOME/.kube`.
+More details on that in the #step-by-step guide below.
```

Comment:
The `#step-by-step` link is not working.

Reply:
Maybe I had some intention to make it a link, but it was left like so. It is linked now.